pkg/virtctl/usbredir/client.go

```diff
@@ -81,6 +83,20 @@ func NewUSBRedirClient(ctx context.Context, address string, stream kvcorev1.Stre
 	return k, nil
 }
 
+// We launch the usbredir client by default.
+func (k *Client) SetLaunchClient(enable bool) {
```

Comment:
Is there actually a point in having a setter for this field? Can't we make it exported instead?

Reply:
The usefulness of the function was only for the second commit, when I wanted to try it out before moving fully to virtctl. I'll rework it.

Reply:
Removed the function, made it exported.